frontend/src/v5/ui/controls/imageWithExtraCount/imageWithExtraCount.component.tsx

```diff
@@ -15,21 +15,21 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Image, ExtraImages, OverlappingContainer } from './commentImage.styles';
+import { Image, ExtraImages, OverlappingContainer } from './imageWithExtraCount.styles';
 
-type CommentImageProps = {
+type ImageWithExtraCountProps = {
 	src: string,
 	className?: string,
 	extraCount?: number,
 	onClick?: () => void,
 };
-export const CommentImage = ({
+export const ImageWithExtraCount = ({
```

Comment:
After opening  the carrousel uploading an image to a comment gets stuck
![error uplloading](https://github.com/user-attachments/assets/686cf7b1-c6d1-40de-83e6-edbe27c9b7bf)